shared/guide/application/chromium/chromium.group

```diff
@@ -0,0 +1,7 @@
+title: Chromium
+
+description: "Chromium is an open-source web browser, powered by WebKit (Blink),\nand developed by Google. Web browsers such\
+    \ as Chromium are used for a number of\nreasons. This section provides settings for configuring Chromium policies to \n\
```

Comment:
What are these backslashes at start and end of line? Do they affect the content? Are they for readability purposes?

Reply:
It's because of how it was laid out in the original XML. In this case there is a leading space aside from indentation on  the following line. Because of how YAML works this space needs to be escaped to be preserved. All of this should disappear in time when we make the move to markdown. The markdown port will require a lot of manual work so I am not sure whether we should solve this particular problem at this stage.